### PR TITLE
Workaround for bug with hidden C3 charts in v0.4.11.

### PR DESCRIPTION
**Pull request by dlabrecq**
Note that an alternative solution is to have patternfly-org depend on C3 v0.4.10.

### CODE REVIEW

File: source/_includes/nav-tabs-pattern.html [hunk 1]
```diff
@@ -14,3 +14,21 @@
...
+                    window.dispatchEvent(new Event('resize'));
+                }
+                // Special case for IE
+                if ($(document).fireEvent) {
+                    $(document).fireEvent('onresize');
+                }
+            }, 100);
+        }
+        $(document).on('shown.bs.tab', 'a[data-toggle="tab"]', resize);
+        $(document).ready(resize);
```
Comment (jeff-phillips-18) [inline source/_includes/nav-tabs-pattern.html]: Are both of these necessary?

Comment (dlabrecq) [inline source/_includes/nav-tabs-pattern.html]: Yes because the user can refresh the web browser. In that case, the previously hidden chart will look bad.

Comment (jeff-phillips-18) [inline source/_includes/nav-tabs-pattern.html]: But I would think the shown event would always fire so no real need for the ready event.

**Comment by dlabrecq** [comment]
You're right, it's not needed. I was also testing $('.nav-tabs').click(), which is why I added it. I'll take it out.

File: source/_includes/nav-tabs-pattern.html (hunk 1 above)
Comment (dlabrecq) [inline source/_includes/nav-tabs-pattern.html]: You're right, it's not needed. I was also testing $('.nav-tabs').click(), which is why I added it. I'll take it out.

**Pull request merged**